nrpy/examples/hydro_without_hydro.py

```diff
@@ -0,0 +1,248 @@
+"""
+Sets up a complete C code project for evolving a TOV solution in a dynamical spacetime with a static fluid ("hydro without hydro").
+
+Author: Leonardo Rosa Werneck
+        wernecklr **at** gmail **dot* com
+"""
+
+#########################################################
+# STEP 1: Import needed Python modules, then set codegen
+#         and compile-time parameters.
+import os
+import shutil
+
+import nrpy.helpers.parallel_codegen as pcg
+import nrpy.infrastructures.BHaH.BHaH_defines_h as Bdefines_h
+import nrpy.infrastructures.BHaH.cmdline_input_and_parfiles as cmdpar
+import nrpy.infrastructures.BHaH.CodeParameters as CPs
+import nrpy.infrastructures.BHaH.CurviBoundaryConditions.CurviBoundaryConditions as cbc
+import nrpy.infrastructures.BHaH.diagnostics.progress_indicator as progress
+import nrpy.infrastructures.BHaH.general_relativity.BSSN_C_codegen_library as BCl
+import nrpy.infrastructures.BHaH.grhd.tov.tov_initial_data_c_codegen as tov
+import nrpy.infrastructures.BHaH.main_c as main
+import nrpy.infrastructures.BHaH.Makefile_helpers as Makefile
+import nrpy.infrastructures.BHaH.numerical_grids_and_timestep as numericalgrids
+import nrpy.infrastructures.BHaH.xx_tofrom_Cart as xxCartxx
+import nrpy.params as par
+from nrpy.helpers.generic import copy_files
+from nrpy.infrastructures.BHaH import (
+    griddata_commondata,
+    rfm_precompute,
+    rfm_wrapper_functions,
+)
+from nrpy.infrastructures.BHaH.MoLtimestepping import MoL
+
+par.set_parval_from_str("Infrastructure", "BHaH")
+
+# Code-generation-time parameters:
+project_name = "hydro_without_hydro"
+IDtype = "tov_interp"
+CoordSystem = "Spherical"
+LapseEvolutionOption = "OnePlusLog"
```

Comment:
The gauge conditions are not consistent with what is needed for the hydro-without-hydro test.

Reply:
Thank you for pointing this out; I didn't realize we used harmonic slicing for the hydro-without-hydro test. This has been fixed.